sources/notation-image-verifier/main.go

```diff
@@ -0,0 +1,46 @@
+// package main provides a simple wrapper of the notation CLI adhering to the
+// containerd image verification plugin interface.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"os/exec"
+)
+
+func main() {
+	var digest = flag.String("digest", "", "image digest to verify")
+	var name = flag.String("name", "", "image name to verify")
+	var _ = flag.String("stdin-media-type", "", "image media type")
+	flag.Parse()
+
+	if *digest == "" || *name == "" {
+		// containerd only captures the stdout of image verification binaries, hence no
+		// write to stderr here (and elsewhere in the program).
+		fmt.Printf("Usage: %s -digest <digest> -name <name>\n", os.Args[0])
+		os.Exit(1)
+	}
+
+	fmt.Printf("verifying image: %s\n", *name)
+
+	// Override the default notation paths to what's packaged in packages/notation
```

Comment:
I'll update this with a direct link to https://notaryproject.dev/docs/user-guides/how-to/directory-structure/ in the comment